android/proguard-rules.pro

```diff
@@ -0,0 +1,17 @@
+# Add project specific ProGuard rules here.
+# By default, the flags in this file are appended to flags specified
+# in [sdk]/tools/proguard/proguard-android.txt
+# You can edit the include path and order by changing the proguardFiles
+# directive in build.gradle.
+#
+# For more details, see
+#   http://developer.android.com/guide/developing/tools/proguard.html
+
+# Add any project specific keep options here:
```

Comment:
all comments in this file?

Reply:
copypasta for the time being. ProGuard is a Java optimizer and obfuscator. i would be extremely surprised if we ended up needing it at all - obfuscation would be pointless since we're OSS, and optimization will likely be equally pointless because the java's only purpose is to load and call into cpp.